packages/terra-i18n/tests/nightwatch/i18n-spec.js

```diff
@@ -12,6 +12,8 @@ const defaultTranslations = enTransaltions;
 
 const screenshot = require('terra-toolkit').screenshot;
 
+const pauseInms = 1000;
```

Comment:
I'm open to adding custom commands assertions, like `waitForText`. What are your thoughts @ryanthemanuel?